client/my-sites/upgrades/domain-management/transfer/index.jsx

```diff
@@ -17,6 +17,15 @@ class Transfer extends React.Component {
 	render() {
 		const slug = this.props.selectedSite.slug,
 			domainName = this.props.selectedDomainName;
+
+		let transferToAnotherUser = null;
```

Comment:
I have a general suggestion for this change. Rather than conditionally storing the component to a variable, I think it'd be a little cleaner and make the markup easier to grok if we just set a constant for the capability here and then use the `{ condition && <Component /> }` trick to conditionally display it.
```js
const canTransferToAnotherUser = ! this.props.selectedSite.options.is_automated_transfer;
[...]
<VerticalNav>
  { canTransferToAnotherUser && <VerticalNavItem /> }
</VerticalNav>
```
Also, are we sure `this.props.selectedSite.options` is all going to be available at this point? If not we should add some checks further up the node.